lib/lang.ts

```diff
@@ -1,17 +1,20 @@
-// @ts-nocheck
-const util = require('util');
-const path = require('path');
-const fs = require('fs-extra');
-const yaml = require('js-yaml');
-const { logger } = require('@hubspot/local-dev-lib/logger');
-const { interpolate } = require('./interpolation');
+import util from 'util';
+import path from 'path';
+// @ts-ignore
+import fs from 'fs-extra';
+// @ts-ignore
+import yaml from 'js-yaml';
```

Comment:
Did you try installing `@typesfs-extra` and `@types/js-yaml`?

Reply:
That worked; thanks, Camden!